tests/test_integrity.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import unittest
+from functools import lru_cache
```

Comment:
did you end up using this somewhere? I don't see a `@lru_cache` annotation in the PR

Reply:
I was going to cache GitHub API calls - but never got around to this. Somehow, flake8 didn't catch this! thanks for reading carefully, I removed it.

Reply:
In cc3b3620 I actually implemented this caching properly. This will be valuable for future additional tests that use GitHub data.